controllers/helpers_test.go

```diff
@@ -101,9 +101,13 @@ func TestGetCloudProviderConfig(t *testing.T) {
 
 	cluster := newCluster("foo")
 	azureCluster := newAzureCluster("bar")
-	azureCluster.Default()
+
+	err := (&infrav1.AzureClusterWebhook{}).Default(context.Background(), azureCluster)
```

Comment:
```suggestion
	g.Expect(&infrav1.AzureClusterWebhook{}).Default(context.Background(), azureCluster).To(Succeed())
```

This can be a one-liner if you like that style. No need to change, just a suggestion.